test/index.test.js

```diff
@@ -11,11 +11,20 @@ describe('AdWords', function() {
   var analytics;
   var options = {
     conversionId: 978352801,
-    events: {
-      signup: '-kGkCJ_TsgcQofXB0gM',
-      login: 'QbThCM_zogcQofXB0gM',
-      play: 'b91fc77f'
-    }
+    labelMap: [
```

Comment:
This is not the data model for a mixed component, it should be

```
labelMap: [
{
  key: 'event name',
  value: {
     ...subconfigs...
  }
}
]
```

you can refer to your slack tests